### PR TITLE
[cubestore] add refreshCredsEveryMinutes parameter to cubestore config

### PR DESCRIPTION
**Pull request by svdimchenko**
I'm getting the following error on cubestore <Error><Code>ExpiredToken</Code><Message>The provided token has expired. This PR should fix it.

### CODE REVIEW

**Comment by lvauvillier** [comment]
Thanks @svdimchenko, i added some comments.

Dont forget also to reference the variable in README.

**Comment by svdimchenko** [comment]
@lvauvillier thanks, added the variable to readme

File: charts/cubestore/values.yaml [hunk 1]
```diff
@@ -136,6 +136,10 @@ cloudStorage:
     ##
     subPath:
 
+    ##The number of minutes after which Cube Store should refresh AWS credentials.
+    ##
+    refresh_creds_every_minutes: 180
```
[the remark this answers is not in the document]
Comment (svdimchenko) [reply charts/cubestore/values.yaml]: sure, fixed

**Comment by lvauvillier** [comment]
Merged. Thanks @svdimchenko!

**Comment by svdimchenko** [comment]
@lvauvillier will you publish the new char version ?

**Comment by lvauvillier** [comment]
Sure. Done!

**Pull request merged**